src/java.base/share/classes/java/lang/System.java

```diff
@@ -768,6 +768,16 @@ public static native void arraycopy(Object src,  int  srcPos,
      * <tr><th scope="row">{@systemProperty native.encoding}</th>
      *     <td>Character encoding name derived from the host environment and/or
      *     the user's settings. Setting this system property has no effect.</td></tr>
+     * <tr><th scope="row">{@systemProperty stdout.encoding}</th>
+     *     <td>Character encoding name for {@link System#out System.out}.
+     *     The property may be set on the command line to the value
+     *     {@code UTF-8}. Setting the property to a value other than {@code UTF-8}
+     *     leads to unspecified behavior.
+     * <tr><th scope="row">{@systemProperty stderr.encoding}</th>
+     *     <td>Character encoding name for {@link System#err System.err}.
+     *     The property may be set on the command line to the value
+     *     {@code UTF-8}. Setting the property to a value other than {@code UTF-8}
+     *     leads to unspecified behavior.
```

Comment:
I think the proposal to introduce two standard properties is good and is consistent with the recently introduced native.encoding. I'm not 100% sure that the sentence "The property may be set on the command line ..." is appropriate for the spec of standard properties. We got away with that for file.encoding in implNote but that isn't spec. I think we may have to replace this with something that says that the Java runtime can be started with the system property set to "UTF-8", starting it with the property set to another value clears to undefined behavior.

Reply:
Thanks, Alan. Modified them as suggested.

Reply:
I think Alan has a typo: "clears" -> "leads"

Reply:
> I think Alan has a typo: "clears" -> "leads"

Apologies, there was a typo in my comment on the PR. I didn't mean for the word "clears" to go into the spec.